crates/protect-ffi/src/encrypt_config.rs

```diff
@@ -0,0 +1,490 @@
+use super::Error;
+use cipherstash_client::schema::{
+    column::{Index, IndexType, TokenFilter, Tokenizer},
+    ColumnConfig, ColumnType,
+};
+use serde::{Deserialize, Serialize};
+use std::{collections::HashMap, str::FromStr};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct Identifier {
+    #[serde(rename = "t")]
+    pub table: String,
+    #[serde(rename = "c")]
+    pub column: String,
+}
+
+impl Identifier {
+    pub fn new<S>(table: S, column: S) -> Self
+    where
+        S: Into<String>,
+    {
+        let table = table.into();
+        let column = column.into();
+
+        Self { table, column }
+    }
+
+    pub fn table(&self) -> &String {
```

Comment:
Should return `&str`

Reply:
Ended up deleting this method (copied it over from Proxy and didn't use it).